.env.example

```
@@ -0,0 +1,111 @@
+WEB__VITE_SUPABASE_PUBLISHABLE_KEY=sb_publishable_ACJWlzQHlZjBrEguHvfOxg_3BJgxAaH
+WEB__VITE_API_BASE_URL=http://localhost:8765/api
+
+# ============================================================
+# MCP  →  mcp/.env   (MCP server)
+# ============================================================
+# Deliberately stays pointed at the HOSTED Supabase project, not the local
+# stack above — the claude.ai OAuth consent dance needs Supabase's hosted
+# OAuth-AS feature + a registered redirect URI, neither set up locally.
+# See mcp/.env.example for the full rationale. Fill in your project ref.
+MCP__SUPABASE_URL=https://<project-ref>.supabase.co
+MCP__SUPABASE_JWKS_URL=https://<project-ref>.supabase.co/auth/v1/.well-known/jwks.json
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MCP__SUPABASE_JWKS_URL key should go before the MCP__SUPABASE_URL key

(UnorderedKey)